lib/main.js

```diff
@@ -301,8 +344,8 @@ function signText(text) {
   }
 
   let certName = selectCert(userCerts, text);
```

Comment:
Maybe we should call this selectCertResult or something similar now (selectedCertInfo?)